Documentation/devel/quickstart-dev.md

```diff
@@ -122,3 +122,21 @@ gofmt -s -w file.go
 [rkt-hacking]: ../hacking.md
 [rkt-dependencies]: ../dependencies.md
 [rkt-tests-readme]: ../../tests/README.md
+
+# Developing rkt on Mac or Windows
+
+In order to develop rkt in non-Linux based operating systems, you can use the provided vagrant `dev` box configuration. 
+In order to launch the dev box use the following commands:
+
+```sh
+go get github.com/coreos/rkt
+export RKT_PATH=$GOPATH/src/github.com/coreos/rkt
```

Comment:
Its there to illustrate where rkt resides in the VM, I'll remove it and add
it as a comment

On Wed, Jul 27, 2016 at 4:02 AM Jonathan Boulle <notifications@github.com>
wrote:

> In Documentation/devel/quickstart-dev.md
> <https://github.com/coreos/rkt/pull/2974#discussion_r72365413>:
>
> > @@ -122,3 +122,21 @@ gofmt -s -w file.go
> >  [rkt-hacking]: ../hacking.md
> >  [rkt-dependencies]: ../dependencies.md
> >  [rkt-tests-readme]: ../../tests/README.md
> > +
> > +# Developing rkt on Mac or Windows
> > +
> > +In order to develop rkt in non-Linux based operating systems, you can use the provided vagrant `dev` box configuration.
> > +In order to launch the dev box use the following commands:
> > +
> > +```sh
> > +go get github.com/coreos/rkt
> > +export RKT_PATH=$GOPATH/src/github.com/coreos/rkt
>
> This env variable seems unnecessary
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coreos/rkt/pull/2974/files/9177403a6f5552d754e9ab06756dd8f9358dd5f3#r72365413>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE1S51B2kjAepJXB268yZ6KVr709-zwZks5qZq4EgaJpZM4JT22a>
> .
>